Kinvey/Kinvey/CustomEndpoint.swift

```diff
@@ -103,11 +134,18 @@ open class CustomEndpoint {
     
     /// Executes a custom endpoint by name and passing the expected parameters.
     @discardableResult
-    open static func execute(_ name: String, params: Params? = nil, client: Client = sharedClient, completionHandler: CompletionHandler<JsonDictionary>? = nil) -> Request {
+    open static func execute(
+        _ name: String,
+        params: Params? = nil,
+        client: Client = sharedClient,
+        requestConfiguration: RequestConfiguration? = nil,
+        completionHandler: CompletionHandler<JsonDictionary>? = nil
+    ) -> Request {
         return execute(
             name,
             params: params,
-            client: client
+            client: client,
```

Comment:
@heyzooi just an idea - a majority of methods in the public API have the `client` parameter. What if we make the `client` a part of requestConfig (defaulting to sharedClient)?